docs/API.md

```diff
@@ -103,11 +103,11 @@ logProcessErrors({
 #### exitOn
 
 _Type_: `string[]`<br> _Value_: array of
-[`"uncaughtException"`](https://nodejs.org/api/process.html#process_event_uncaughtexception),
-[`"unhandledRejection"`](https://nodejs.org/api/process.html#process_event_unhandledrejection),
-[`"rejectionHandled"`](https://nodejs.org/api/process.html#process_event_rejectionhandled),
-[`"multipleResolves"`](https://nodejs.org/api/process.html#process_event_multipleresolves)
-or [`"warning"`](https://nodejs.org/api/process.html#process_event_warning)<br>
+[`'uncaughtException'`](https://nodejs.org/api/process.html#process_event_uncaughtexception),
+[`'unhandledRejection'`](https://nodejs.org/api/process.html#process_event_unhandledrejection),
+[`'rejectionHandled'`](https://nodejs.org/api/process.html#process_event_rejectionhandled),
+[`'multipleResolves'`](https://nodejs.org/api/process.html#process_event_multipleresolves)
+or [`'warning'`](https://nodejs.org/api/process.html#process_event_warning)<br>
 _Default_: `["uncaughtException"]`
 
 Which process errors should trigger `process.exit(1)`:
```

Comment:
You missed the ones there.

Reply:
Sorry, I use a RegExp to find the double quote (with `) so I miss them